kamelet-support/src/main/java/io/kaoto/backend/model/deployment/kamelet/expression/Expression.java

```diff
@@ -74,7 +74,6 @@ protected void assignAttribute(final Parameter parameter) {
                 this.setName(parameter.getValue().toString());
                 break;
             default:
```

Comment:
This is just adding noise to the log. OF course we don't know about this property.